ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -421,10 +456,18 @@ const OmimSegments = styled(Segment.Group).attrs({ size: 'tiny', horizontal: tru
   }
 `
 
-const getDetailSections = (configs, gene, compact, labelProps, rnaSeqData) => configs.map(
+const getDetailSections = (configs, gene, compact, labelProps, individualGeneData) => configs.map(
   ({ showDetails, detailsDisplay, ...sectionConfig }) => (
-    { ...sectionConfig, detail: showDetails(gene, rnaSeqData) && detailsDisplay(gene, rnaSeqData) }),
-).filter(({ detail }) => detail).map(({ detail, expandedDisplay, ...sectionConfig }) => (
+    {
+      ...sectionConfig,
+      detail: showDetails(gene, individualGeneData) && detailsDisplay(gene, individualGeneData),
+    }),
+).reduce((acc, config) => (Array.isArray(config.detail) ?
+  [
+    ...acc,
+    ...config.detail.map(detail => ({ ...config, ...detail })),
+  ] : (config.detail && [...acc, config]) || acc),
```

Comment:
`x && y || z` is not readable/standard syntax - that should be a ternary expression `x ? y : z`. If it is throwing an error about nested ternarys, the solution is either to  not do this in one line or to add in an explicit call to `filter`, not to replace the nested ternary with an unreadable line of code

Reply:
Nested ternaries throw an error. I will add a filter.